stablehlo/reference/Ops.cpp

```diff
@@ -456,18 +430,18 @@ SmallVector<Tensor> eval(Region &region, ArrayRef<Tensor> args, Scope *parent) {
       scope.add(op.getResults(), {runtimeResult});
     } else if (auto dynamicSliceOp = dyn_cast<DynamicSliceOp>(op)) {
       Tensor runtimeOperand = scope.find(dynamicSliceOp.getOperand());
-      auto runtimeStartIndices = scope.find(dynamicSliceOp.getStartIndices());
-      auto runtimeSliceSizes =
-          llvm::to_vector(dynamicSliceOp.getSliceSizes().getValues<int64_t>());
+      Index runtimeStartIndices =
+          evalIndices(scope.find(dynamicSliceOp.getStartIndices()));
```

Comment:
I noticed that this changes the API of `evalDynamicSliceOp` and `evalDynamicUpdateSliceOp`. In the StableHLO opset, the corresponding ops are taking tensors not tensor constants, so how about we stick to the existing API which also takes tensors and call evalIndices inside the corresponding functions?

Reply:
Sure.